lte/gateway/c/core/oai/tasks/ngap/include/ngap_client_servicer.h

```diff
@@ -24,9 +24,16 @@ extern "C" {
 #endif
 #include <memory.h>
 #include <string.h>
+#include "lte/protos/oai/s1ap_state.pb.h"
+#include "lte/gateway/c/core/oai/include/map.h"
 
 namespace magma5g {
 
+// Ngap-Map Declarations:
+// Map- Key: string, Data: S1apImsiMap
+typedef magma::map_s<std::string, magma::lte::oai::S1apImsiMap>
```

Comment:
@sreedharkumartn, 
   this map should be removed, ngap_client_servicer.h should hold the key and serialized string of S1apImsiMap.Proto
   this map should be like Radis DB table for reading/write.

Reply:
Addressed Review Comment